### PR TITLE
Sync: Update widgets so that we pass in the Widget name and Sidebar name when ever possible

### PR DESCRIPTION
**Pull request by enejb**
Previously we were no passing in the widget name and the sidebar name. but instead were just passing in the ids. Which are not enough to build a user friendly Activity log. 

We are adding the names the way we are so that they are backwards compatible if someone else is using the do_actions already. 

#### Changes proposed in this Pull Request:
* In this pr we are adding the names of the sidebar and the widget whenever possible 

#### Testing instructions:
* Do the test pass? Do it pass when testing things manually

### CODE REVIEW

**Comment by lezama** [comment]
tests are not behaving :)

File: sync/class.jetpack-sync-module-themes.php [hunk 1]
```diff
@@ -354,9 +376,10 @@ function sync_sidebar_widgets_actions( $old_value, $new_value ) {
 			 *
 			 * @since 4.9.0
 			 *
-			 * @param array $sidebar, Sidebar id got changed
+			 * @param array $moved_to_inactive, Array of widgets id that moved to inactive id got changed
+			 * @param array $moved_to_inactive, Array of widgets names that moved to inactive id got changed Since 5.0.0
 			 */
-			do_action( 'jetpack_widget_moved_to_inactive', $moved_to_inactive );
+			do_action( 'jetpack_widget_moved_to_inactive', $moved_to_inactive, array_map( array( $this, 'get_widget_name' ), $moved_to_inactive ) );
```
Comment (gititon) [inline sync/class.jetpack-sync-module-themes.php]: Consider renaming $moved_to_inactive to $moved_to_inactive_ids
Consider reducing `array_map( array( $this, 'get_widget_name' ), $moved_to_inactive ) )` to a variable called $moved_to_inactive_names and passing it as an argument to do_action

**Comment by enejb** [comment]
Thanks for the review, fixed with your suggestion @gititon

File: sync/class.jetpack-sync-module-themes.php [hunk 2]
```diff
@@ -245,23 +246,39 @@ function expand_theme_data() {
...
 			 * @since 4.9.0
 			 *
 			 * @param string $sidebar, Sidebar id got changed
 			 * @param string $added_widget, Widget id got added
+			 * @param string $sidebar_name, Sidebar id got changed Since 5.0.0
+			 * @param string $added_widget_name, Widget id got added Since 5.0.0
+			 *
 			 */
-			do_action( 'jetpack_widget_added', $sidebar, $added_widget );
+			do_action( 'jetpack_widget_added', $sidebar, $added_widget,  $sidebar_name, $added_widget_name ); // $this->get_sidebar_name( $sidebar ), $this->get_widget_name( $added_widget ) );
```
Comment (roccotripaldi) [inline sync/class.jetpack-sync-module-themes.php]: is the comment needed here?

Comment (enejb) [inline sync/class.jetpack-sync-module-themes.php]: Good catch!

**Comment by dereksmart** [comment]
Two reviews already... merging.

**Pull request merged**